api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/IntensiveMonitoringCohortQueries.java

```diff
@@ -1658,30 +1658,29 @@ public CohortDefinition getMI15F() {
     map.put("1305", hivMetadata.getHivViralLoadQualitative().getConceptId());
 
     String query =
-            ""
-                    + "SELECT p.patient_id "
-                    + "FROM   patient p "
-                    + "       INNER JOIN encounter ee ON ee.patient_id = p.patient_id "
-                    + "       INNER JOIN obs oo ON oo.encounter_id = ee.encounter_id "
-                    + "WHERE  p.voided = 0 "
-                    + "       AND ee.voided = 0 "
-                    + "       AND oo.voided = 0 "
-                    + "       AND ee.location_id = :location "
-                    + "       AND ee.encounter_type = ${6} "
-                    + "       AND oo.concept_id = ${1695} "
-                    + "       AND oo.value_numeric <= 200 "
-                    + "       AND ee.encounter_datetime <= :endDate "
-                    + "       AND NOT EXISTS (SELECT e.encounter_id "
-                    + "                       FROM   encounter e    "
-                    + "                              INNER JOIN obs o ON o.encounter_id = e.encounter_id "
-                    + "                       WHERE  e.encounter_type = ${6} "
-                    + "                              AND e.location_id = :location "
-                    + "                              AND o.concept_id IN( ${856}, ${1305} ) "
-                    + "                              AND e.patient_id = p.patient_id "
-                    + "                              AND e.encounter_datetime <= :endDate "
-                    + "                              AND e.voided = 0 "
-                    + "                              AND o.voided = 0)"
-                    + " GROUP BY p.patient_id         ";
+              "SELECT p.patient_id "
+            + "FROM   patient p "
+            + "       INNER JOIN encounter ee ON ee.patient_id = p.patient_id "
+            + "       INNER JOIN obs oo ON oo.encounter_id = ee.encounter_id "
+            + "WHERE  p.voided = 0 "
+            + "       AND ee.voided = 0 "
+            + "       AND oo.voided = 0 "
+            + "       AND ee.location_id = :location "
+            + "       AND ee.encounter_type = ${6} "
+            + "       AND oo.concept_id = ${1695} "
+            + "       AND oo.value_numeric <= 200 "
+            + "       AND ee.encounter_datetime <= :endDate "
+            + "       AND NOT EXISTS (SELECT e.encounter_id "
```

Comment:
What are we excluding here? encounters? NOT sure am getting it right, can **NOT IN**